internal/build/custom_builder.go

```diff
@@ -107,20 +106,21 @@ func (b *CustomBuilder) Build(ctx context.Context, refs container.RefSet,
 			l.Infof("  %s", v)
 		}
 	}
-	cmd.Env = append(os.Environ(), extraEnvVars...)
-	err = imagemap.InjectIntoLocalEnv(cmd, spec.ImageMaps, imageMaps)
+	cmd.Spec.Env = append(cmd.Spec.Env, extraEnvVars...)
```

Comment:
Will the spec already have the `os.Environ()`?

AFAICT we don't add it before execing in `ForceRun` -> https://github.com/nicks/tilt/blob/cfca55cae493dce8616a0ebf986d10e9922eec0a/internal/controllers/core/cmd/controller.go#L389-L399